### PR TITLE
Update swagger schemas - Closes #5098

### LINKED ISSUE #5098: Update swagger schemas
### Description

Update `swagger.yml` to match new schema after changes in #5083 

### Motivation

Changes were made to the schema of some responses and needs to be updated

### Acceptance Criteria

- Schema is updated to match response of HTTP controllers for `/account`, 
- Schema is updated to match response of HTTP controllers for `/delegates`
- Schema is updated to match response of HTTP controllers for `/votes`
- Schema is updated to match response of HTTP controllers for `/voters`

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

This PR resolves #5098

### How was it solved?

By updating the swagger schema

### How was it tested?

Compare swagger vs call to endpoints described in the issue

### CODE REVIEW

File: framework/src/modules/http_api/schema/swagger.yml [hunk 1]
```diff
@@ -2208,13 +2220,13 @@ definitions:
...
           It is unique and cannot be changed later.
-      voteWeight:
+      totalVotesReceived:
         type: string
         example: 1081560729258
         description: |
-          The voters weight of the delegate.
-          Represents the total amount of Lisk (in Beddows), that the delegates voters own.
-          The voters weight decides which rank the delegate gets in relation to the other delegates and their voters weights.
+          The total votes received by the delegate.
```
Comment (shuse2) [inline framework/src/modules/http_api/schema/swagger.yml:2227]:
```suggestion
          The total votes received of the delegate.
```
?

File: framework/src/modules/http_api/schema/swagger.yml [hunk 2]
```diff
@@ -2208,13 +2220,13 @@ definitions:
...
+          Represents the total amount of Lisk (in Beddows), that the delegates voters voted this delegate.
+          It takes into account the calculation of self versus received votes.
```
Comment (shuse2) [inline framework/src/modules/http_api/schema/swagger.yml:2229]: I think this line is not needed

File: framework/src/modules/http_api/schema/swagger.yml [hunk 3]
```diff
@@ -2118,6 +2118,8 @@ definitions:
       - publicKey
       - balance
       - nonce
+      - votes
+      - delegate
```
Comment (shuse2) [inline framework/src/modules/http_api/schema/swagger.yml:2122]: I think `keys` and `unlocking` is missing here and in properties

File: framework/src/modules/http_api/schema/swagger.yml [hunk 4]
```diff
@@ -2227,18 +2239,45 @@ definitions:
...
+        type: string
+        example: 154
+        description: The current nonce associated to account for transaction processing.
+      asset:
+        type: object
+        description: The asset object for the account.
+      keys:
+        type: object
+        description: Multisignature information for the account.
+      votes:
```
Comment (shuse2) [inline framework/src/modules/http_api/schema/swagger.yml:2270]: `unlocking` is missing here

File: framework/src/modules/http_api/schema/swagger.yml [hunk 5]
```diff
@@ -2145,8 +2147,19 @@ definitions:
...
+        items:
+          $ref: '#/definitions/Vote'
       delegate:
         $ref: '#/definitions/Delegate'
+      isDelegate:
+          type: boolean
+          description: The value indicating if the account is a deleage or not.
+      username:
+          type: string
+          example: DelegateABC
```
Comment (shuse2) [inline framework/src/modules/http_api/schema/swagger.yml:2161]: example should be in lower case

File: framework/src/modules/http_api/schema/swagger.yml [hunk 6]
```diff
@@ -2145,8 +2147,19 @@ definitions:
...
+          description: If the account is a delegate it displays the username for it.
```
Comment (shuse2) [inline framework/src/modules/http_api/schema/swagger.yml:2162]:
```suggestion
          description: If the account is a delegate, it displays the username for it.
```

File: framework/src/modules/http_api/schema/swagger.yml (hunk 1 above)
Comment (pablitovicente) [reply framework/src/modules/http_api/schema/swagger.yml:2227]: @shuse2 are you sure `of` is correct? this are the votes the delegate received

Comment (shuse2) [reply framework/src/modules/http_api/schema/swagger.yml:2227]: Im not sure =)
but those votes are not necessarily  voted by delegate, but it sounds like it

File: framework/src/modules/http_api/schema/swagger.yml (hunk 3 above)
Comment (pablitovicente) [reply framework/src/modules/http_api/schema/swagger.yml:2122]: Added (given that the code deletes the `keys` property if empty I did not added it as required)

File: framework/src/modules/http_api/schema/swagger.yml (hunk 5 above)
Comment (pablitovicente) [reply framework/src/modules/http_api/schema/swagger.yml:2161]: updated

File: framework/src/modules/http_api/schema/swagger.yml (hunk 2 above)
Comment (pablitovicente) [reply framework/src/modules/http_api/schema/swagger.yml:2229]: removed

File: framework/src/modules/http_api/schema/swagger.yml (hunk 4 above)
Comment (pablitovicente) [reply framework/src/modules/http_api/schema/swagger.yml:2270]: added

**Comment by pablitovicente** [comment]
> delegate

Yeah probably not related to this PR. I do think `delegate` should include username in the database too but for some reason it was left out of it. So maybe create a different issue to discuss this matters?

**Pull request merged**